apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -488,6 +565,44 @@ export default Vue.extend({
             this.copyCell(cell)
           }
         },
+        {
+          label: `<x-menuitem><x-label>Edit row as JSON</x-label></x-menuitem>`,
```

Comment:
I really don't know what we should say here, something like 'Edit in Editor' but I don't like how that sounds :sweat_smile: The main thing is you are editing a cell, not a row, and I don't think it should be language specific here. The modal editor will ideally autodetect the type of content being edited.

Reply:
Hey! We can fix copy once it's merged. That's easy for us to do, mostly let's just check functionality, which looks like it's doing the right thing.